tests/test_density_GaussianDensity.py

```diff
@@ -103,6 +103,38 @@ def test_sum_3d(self):
         # This has discretization error as well as single-precision error
         assert np.isclose(np.sum(gd.density), 1, atol=1e-4)
 
+    def test_sum_val_2d(self):
+        # Ensure that the Gaussian convolution sums to average of the values
+        width = 100
+        r_max = 49
+        sigma = 10
+        num_points = 1
+        box_size = width
+        system = freud.data.make_random_system(
+            box_size, num_points, is2D=True)
+        values = np.random.rand(num_points)
+
+        gd = freud.density.GaussianDensity(width, r_max, sigma)
+        gd.compute(system, values)
+        # This has discretization error as well as single-precision error
+        assert np.isclose(np.sum(gd.density), np.mean(values), atol=1e-4)
```

Comment:
I think this only works because `num_points = 1`. I think the total should be `num_points * np.mean(values)`, otherwise written as `np.sum(values)`.
```suggestion
        assert np.isclose(np.sum(gd.density), np.sum(values), atol=1e-4)
```